help/en/html/hardware/mqtt/index.shtml

```diff
@@ -292,6 +292,34 @@
         their aspect when changed.  These do not receive anything from the layout.
         </li>
 
+        <li><strong>Track Power</strong>: JRMI sends Track Power ON/OFF commands on the "Power
+        send topic" and receives the track power status on the "Power receive topic".
+        </li>
+
+        <li><strong>Throttle Control</strong>: JRMI supports controlling cabs over MQTT.  The protocol is
+        controller agnostic though the internal design of JRMI imposes some compliance with DCC standards.
+        <br>Using a Throttle Control over MQTT provides for:
+        <ul>
+        <li><strong>Throttle</strong>: The speed setting as a 0-100 percentage value is sent on the
+        "Throttle send topic" and received on the "Throttle receive topic".</li>
+        <li><strong>Direction</strong>: The throttle direction is sent on the "Direction send topic" and
+        received on the "Direction receive topic".  Direction values are "FORWARD" and "REVERSE", and "STOP"
+        for an emergency stop.</li>
+        <li><strong>Function</strong>: Function states are sent on the "Function send topic" and received
+        on the "Function receive topic".  The string "$function" within each topic is substituted with
+        the number of the function.Function state values are "ON" and "OFF".</li>
+        <li><strong>Consist</strong>: MQTT supports Command Station Consists in JRMI using the standard
```

Comment:
Typo: JRMI instead of JMRI 2 places - a very minor thing!

Reply:
Fixed